q2_qemistree/_plot.py

```diff
@@ -127,7 +127,8 @@ def format_barplots(table: biom.Table):
 def plot(output_dir: str, tree: NewickFormat, feature_metadata: pd.DataFrame,
          category: str = 'class', ms2_label: bool = False,
          color_palette: str = 'Dark2', parent_mz: bool = True,
-         grouped_table: biom.Table = None) -> None:
+         grouped_table: biom.Table = None,
+         normalize_features: bool = True) -> None:
```

Comment:
This is leading to an error in the CLI, you need to update plugin_setup.py as well:

```
(qiime2-dev) yoshikivazquezbaeza:q2-qemistree@gfop_fixes$ qiime dev refresh-cache
QIIME is caching your current deployment for improved performance. This may take a few moments and should only happen once per deployment.
Traceback (most recent call last):
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/bin/qiime", line 11, in <module>
    sys.exit(qiime())
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/builtin/dev.py", line 31, in refresh_cache
    import q2cli.core.cache
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/core/cache.py", line 404, in <module>
    CACHE = DeploymentCache()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/core/cache.py", line 61, in __init__
    self._state = self._get_cached_state(refresh=refresh)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/core/cache.py", line 107, in _get_cached_state
    self._cache_current_state(current_requirements)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/core/cache.py", line 200, in _cache_current_state
    state = self._get_current_state()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/q2cli/core/cache.py", line 238, in _get_current_state
    plugin_manager = qiime2.sdk.PluginManager()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/sdk/plugin_manager.py", line 44, in __new__
    self._init()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/sdk/plugin_manager.py", line 59, in _init
    plugin = entry_point.load()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2444, in load
    return self.resolve()
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2450, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/yoshikivazquezbaeza/git_sw/q2-qemistree/q2_qemistree/plugin_setup.py", line 280, in <module>
    citations=[citations['letunic2019itol']])
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/plugin/plugin.py", line 260, in register_function
    citations, deprecated)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/sdk/action.py", line 463, in _init
    parameter_descriptions)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/core/type/signature.py", line 457, in __init__
    output_descriptions)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/core/type/signature.py", line 115, in __init__
    output_descriptions)
  File "/Users/yoshikivazquezbaeza/miniconda3/envs/qiime2-dev/lib/python3.6/site-packages/qiime2/core/type/signature.py", line 189, in _parse_signature
    " %r" % name)
TypeError: Parameter in callable without QIIME type: 'normalize_features'
```

Reply:
That's right, fixed it! thanks 👍